$OpenBSD: patch-addons_Socket_CMakeLists_txt,v 1.1.1.1 2012/12/17 23:55:28 jeremy Exp $

The Socket extension needs to link to two libraries, and I'm
not sure how to do that with CMake arguments, so this patch
is used for the second library.

--- addons/Socket/CMakeLists.txt.orig	Wed Oct 10 19:19:48 2012
+++ addons/Socket/CMakeLists.txt	Wed Oct 10 19:20:03 2012
@@ -48,7 +48,7 @@ if(EVENT_FOUND)
 		)
 	ENDIF(NOT MSVC AND NOT MINGW)
 
-	set(LIBS "")
+	set(LIBS "-levent_extra")
 	
 	if(MSVC)
 		set(LIBS "Ws2_32.lib Iphlpapi.lib")
